doc/libdevq.3

```diff
@@ -38,7 +38,7 @@ Portable query and monitoring library (libdevq, -ldevq)
 .Fc
 .Ft const char *
 .Fo devq_device_get_devpath_from_fd
-.Fa "struct devq_device *device"
+.Fa "int fd, char *path, size_t *path_len"
```

Comment:
You need to put every argument on a new line with .Fa in front.Like it was done with devq_device_get_pciid_from_fd().